src/js/structs/Service.js

```diff
@@ -81,12 +86,32 @@ module.exports = class Service extends Item {
     };
   }
 
+  getStatus() {
+    let {tasksRunning} = this.getTasksSummary();
+    let deployments = this.getDeployments();
+
+    if (deployments.length > 0) {
+      return ServiceStatus.DEPLOYING.displayName;
+    }
+
+    if (tasksRunning > 0) {
+      return ServiceStatus.RUNNING.displayName;
+    }
+
+    let instances = this.getInstances();
+    if (instances === 0) {
+      return ServiceStatus.SUSPENDED.displayName;
+    }
+  }
+
   getTasksSummary() {
     return {
       tasksHealthy: this.get('tasksHealthy'),
       tasksRunning: this.get('tasksRunning'),
       tasksStaged: this.get('tasksStaged'),
-      tasksUnhealthy: this.get('tasksUnhealthy')
+      tasksUnhealthy: this.get('tasksUnhealthy'),
+      tasksUnknown: this.get('tasksRunning') -
```

Comment:
I don't see any reason not to add aggregated data, as long as it is consistent and *useful*. In my humble opinion are structs (at least in our case) meant to typecast data, provide an abstraction layer, and a nice and easy to use interface.